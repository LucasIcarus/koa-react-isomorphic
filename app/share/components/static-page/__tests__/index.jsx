import React from "react";
import { shallow } from "enzyme";
import StaticPage from "../index";

describe("Component: StaticPage", () => {
  let component;

  beforeEach(() => {
    component = shallow(<StaticPage />);
  });

  it("should render component", () => {
    expect(component).toMatchSnapshot();
  });
});
