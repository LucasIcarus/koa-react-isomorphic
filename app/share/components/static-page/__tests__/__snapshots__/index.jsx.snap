// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: StaticPage should render component 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-md-12"
    >
      <h1>
        Static Page
      </h1>
      <Connect(BaseLink)
        to="/"
      >
        Back to Home page
      </Connect(BaseLink)>
    </div>
  </div>
</div>
`;
